app/views/blog/posts/2011-07-11-reset.html

```diff
@@ -65,10 +66,10 @@ <h3 class="subtitle">
 for each file in the HEAD snapshot:</p>
 
 <pre>
-$ cat .git/HEAD 
+$ cat .git/HEAD
```

Comment:
These bits don't look relevant to the purpose of the PR, but I'm OK with dropping trailing whitespace.